handbook/engineering/hiring/software-engineer-pairing-exercise.md

```diff
@@ -1,13 +1,46 @@
 # Pairing exercise
 
-We ask candidates to spend an hour pairing with an engineer on the team they are applying to. **The pairing exercise is not used for technical evaluation.** That's what the [async coding exercise](software-engineer-coding-exercise.md) is for.
+We ask candidates to spend an hour pairing with an engineer on the team they are applying to. **The pairing exercise is not used for technical evaluation.** That's what the [async coding exercise](software-engineer-coding-exercise.md) (which comes first in the interview process) is for.
+
+The assumption is that any candidate who gets to this stage meets the technical requirements for the job in question.
+
+All pairing sessions should include a third engineer in the room as an observer. As such, they can stay muted for the bulk of the interview. Having another person
+in the room provides a more complete picture of the candidate and helps to reduce bias.
 
 ## What we're looking for
 
+At a high level, this is what we are evaluating our candidates for:
+
 - Clear explanations of ideas and decisions.
 - Frequent communication while pairing.
 - Willingness to drive, while the other engineer in the loop.
 - Ability to let the interviewer drive while staying engaged.
 - Flexibility in considering other ideas and approaches, and in making changes to the code.
 - Ability to break down a problem and decide on an approach as a pair.
 - Healthy responses to feedback.
+
+## Before the interview
+
+The engineer leading the pairing exercise should choose a relatively easy task to work on. An ideal exercise should:
+
+- be easy to explain
+- be language agnostic, so the candidate can use whatever language/tools they are most comfortable with
```

Comment:
I am confused, are we asking the interviewer to come up with their own unique interview question? (this seems like a bad idea)

Or...are we asking the engineer to pair with the candidate on some real task they are actually working on?

Reply:
In the past, I have asked the engineer leading the interview to come up with an exercise (that I approved according to the given criteria), and then made sure to test it out with them.

The idea is that this happens only once, because they reuse the same exercise each time.

(I have several suggestions for these exercises if the interviewer would like suggestions, but I didn't want to put them in this doc.)

Reply:
It would make me more comfortable if we did that work right now and documented the results here.